src/modinv32.h

```diff
@@ -35,4 +35,8 @@ static void secp256k1_modinv32_var(secp256k1_modinv32_signed30 *x, const secp256
 /* Same as secp256k1_modinv32_var, but constant time in x (not in the modulus). */
 static void secp256k1_modinv32(secp256k1_modinv32_signed30 *x, const secp256k1_modinv32_modinfo *modinfo);
 
+/* Compute the Jacobi symbol for (x | modinfo->modulus). Either x must be 0, or x must be coprime with
```

Comment:
Nit: The jacobi symbol (0|1) is commonly defined to be 1 (see e.g. [wikipedia](https://en.wikipedia.org/wiki/Jacobi_symbol)), but `secp256k1_jacobi_maybe_var` returns 0 in that case. I think it would be valuable to document `secp256k1_jacobi_maybe_var`'s behavior in this edge case.

Reply:
That's related to what I said there: https://github.com/bitcoin-core/secp256k1/pull/979#discussion_r1062661604. Though I don't understand why I said there that I don't see why the output is trivial. It's trivial, of course. Maybe what I meant is that I don't see immediately why the output is correct, which matches your comment here, @jonasnick.

Perhaps it's better to simply exclude (0|1) as a valid input, and then document and VERIFY_CHECK that. That makes it easier to recognize this corner case in future code. (And I feel it's hard to predict what definition of jac(0|1) future code would expect to work. It's better to make callers remind of the corner case explicitly.)